source/stylesheets/globals/imports/finescss/_fine_mixins.scss

```diff
@@ -18,6 +18,96 @@
   list-style: none;
 }
 
+// Create iOS style text underlines
+// Comes in one mixin containing two additional mixins
+// each nested mixin depends on function/keyframe helper
+  // | underline-link
+    // |- underline-link-shadow
+    //  |- function: skew-it
+    // |- underline-transition
+    //  |- keyframe: underline-width
+// @include underline-link(white, black);
+// @include underline-link(white, black, false);
+
+@mixin underline-link-shadow($color) {
+  text-shadow:
+  skew-it(1) 0 $color, skew-it(-1) 0 $color,
```

Comment:
LDT, indentation for all declarations under `text-shadow` should be one level (2 spaces) further.